src/tools/cgnscheck.c

```diff
@@ -27,6 +27,8 @@
 
 #define USE_MID_NODES
 
+#define min(A,B) A <= B ? A : B;
```

Comment:
min and max should be already located in cgns_header.h

Reply:
Mistakes. I will correct in the next commit